readme.didact.md

```diff
@@ -151,12 +161,12 @@ that Camel K integrations will use to publish events or subscribe to it in order
 types are available. Subscribers of the eventing broker are Knative serving services, that can scale down to zero when no
 events are available for them.
 
-To enable the eventing broker, we create a `default` broker in the current namespace using namespace labeling:
+To enable the eventing broker, we create a `default` broker in the current namespace using the Knative CLI:
 
 ```
-oc label namespace camel-knative knative-eventing-injection=enabled
```

Comment:
`oc label namespace xxxxxxxxx eventing.knative.dev/injection=enabled `  will still work

Only the `knative-eventing-injection=enabled` syntax is deprecated, and will be removed SOON!

Reply:
Thanks @matzew, does it need the "sugar-controller"? Is it enabled on OpenShift serverless?